Services/VIWIS/classes/class.genericFeedback.php

```diff
@@ -0,0 +1,25 @@
+<?php
+require_once 'Services/VIWIS/interfaces/interface.generalDBIndependentFeedback.php';
+class genericFeedback implements generalDBIndependentFeedback {
```

Comment:
Ok. Können wir den Klassennamen bitte trotzdem, wie sonst auch, in CamelCase schreiben?